android/src/main/java/com/cardio/RNCardIOModule.java

```diff
@@ -39,6 +39,7 @@ public void scanCard(ReadableMap config, Promise promise) {
     Intent scanIntent = new Intent(activity, CardIOActivity.class);
     scanIntent.putExtra(CardIOActivity.EXTRA_REQUIRE_EXPIRY, true);
     scanIntent.putExtra(CardIOActivity.EXTRA_REQUIRE_CVV, true);
+    scanIntent.putExtra(CardIOActivity.EXTRA_KEEP_APPLICATION_THEME, true);
```

Comment:
You set in the README the default value is `false` but here it seems to be `true`.